package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-time-select",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Can you change to 1.0.2 please and add a changelog? I published 1.0.1 the other day trying to debug something, and won't be able to publish over it. (1.0.1 fixed naming of a `propType` being inconsistent with a `defaultProp`)